README.md

```diff
@@ -188,6 +192,25 @@ o-tracking will
 * Automatically pickup ftsession from cookies for you.
 * Page events automatically pick up the url and the referrer.
 * Click events [initalised as above](#usage), will populate a `domPathTokens` property. If the clicked element has the `data-trackable` attribute, sibling elements will also be included within `domPathTokens`.
+* Component view events will be fired with elements set `data-o-tracking-view` attribute or elements collected by a custom selector in `opts`. They will populate a `domPathTokens` as well as Click events. You could pass a custom category, and also a callback[`getContextData`] to collect data for events.  
+:warning: This feature requires `window.IntersectionObserver` in order to track the events  
+:warning: `getContextData` should be a function which returns `{Object}`
+	```js
+	const opts = {
+		category: 'audio', // default: 'component'
+		selector: '.o-teaser__audio', // default: '[data-o-tracking-view]'
+		getContextData: () => {  // default: null
```

Comment:
If I understand correctly, the callback function accepts the viewed element as an argument? It would be great to document that here.

Reply:
👍